Install fails shellcheck

New to nix, but I ran into this error when trying to install the module on my system. The logs just showed the same error. Same error on both the zhaofengli-wip/nix-homebrew and zhaofengli/nix-homebrew repositories (not sure which is more correct). Using a mac with Sonoma 14.6.1 with an M1 chip. Let me know if you need any other environment info.

error:
```
• Updated input 'nix-homebrew':
    'github:zhaofengli-wip/nix-homebrew/5eb00c5e0141be799d4708136dc036b23b2f9b4a?narHash=sha256-7fW9uTLG6G9pJo9c4uenwUi%2BWT2jUz4jAp7Xh0EnYxU%3D' (2024-11-19)
  → 'github:zhaofengli/nix-homebrew/5eb00c5e0141be799d4708136dc036b23b2f9b4a?narHash=sha256-7fW9uTLG6G9pJo9c4uenwUi%2BWT2jUz4jAp7Xh0EnYxU%3D' (2024-11-19)
error: builder for '/nix/store/xzabffx8fjph3v31j8q4f7vbss903m9r-darwin-system-25.05.20241118.5083ec8+darwin5.61cee20.drv' failed with exit code 1;
       last 7 log lines:
       >
       > In /nix/store/a4gn5vf5m6ibwhgnmjr1zdsrcwxn7mpj-darwin-system-25.05.20241118.5083ec8+darwin5.61cee20/activate-user line 61:
       > INSTALLING_HOMEBREW=1
       > ^-----------------^ SC2034 (warning): INSTALLING_HOMEBREW appears unused. Verify use (or export if used externally).
       >
       > For more information:
       >   https://www.shellcheck.net/wiki/SC2034 -- INSTALLING_HOMEBREW appears unuse...
       For full logs, run 'nix log /nix/store/xzabffx8fjph3v31j8q4f7vbss903m9r-darwin-system-25.05.20241118.5083ec8+darwin5.61cee20.drv'.
error: Recipe `reload` failed on line 2 with exit code 1
```

flake.nix:
```
{
  description = "cagroleau nix-darwin system flake";

  inputs = {
    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
    nix-darwin.url = "github:LnL7/nix-darwin";
    nix-darwin.inputs.nixpkgs.follows = "nixpkgs";
    nix-homebrew.url = "github:zhaofengli/nix-homebrew";
  };

  outputs = inputs@{ self, nix-darwin, nixpkgs, nix-homebrew }:
  let
    configuration = { pkgs, ... }: {
      # List packages installed in system profile. To search by name, run:
      # $ nix-env -qaP | grep wget
      environment.systemPackages =
        [
          pkgs.neovim
          (...)
        ];

      # Necessary for using flakes on this system.
      nix.settings.experimental-features = "nix-command flakes";

      # Enable alternative shell support in nix-darwin.
      # programs.fish.enable = true;

      # Set Git commit hash for darwin-version.
      system.configurationRevision = self.rev or self.dirtyRev or null;

      # Used for backwards compatibility, please read the changelog before changing.
      # $ darwin-rebuild changelog
      system.stateVersion = 5;

      # The platform the configuration will be used on.
      nixpkgs.hostPlatform = "aarch64-darwin";
    };
  in
  {
    # Build darwin flake using:
    # $ darwin-rebuild build --flake .#simple
    darwinConfigurations."macbook" = nix-darwin.lib.darwinSystem {
      modules = [ 
        configuration
        nix-homebrew.darwinModules.nix-homebrew
        {
          nix-homebrew = {
            enable = true;
            enableRosetta = true;

            user = "username";

            autoMigrate = true;
          };
        }
      ];
    };

    # Expose the package set, including overlays, for convenience.
    darwinPackages = self.darwinConfigurations."macbook".pkgs;
  };
}
```

--- a/modules/default.nix
+++ b/modules/default.nix
@@ -502,8 +502,8 @@ in {
 
     # disable the install homebrew check
     # see https://github.com/LnL7/nix-darwin/pull/1178 and https://github.com/zhaofengli/nix-homebrew/issues/45
-    system.checks.text = lib.mkBefore ''
+    system.checks.text = lib.mkIf config.homebrew.enable (lib.mkBefore ''
       INSTALLING_HOMEBREW=1
-    '';
+    '');
   };
 }

